lrc/Pos/Lrc/Fts.hs

```diff
@@ -237,3 +262,19 @@ followTheSatoshiM (SharedSeed seed) totalCoins = do
                 findLeader coinIndex
 
     nextStakeholder = fromMaybe (error "followTheSatoshiM: indices out of range") <$> await
+
+
+-- | A pure version of `followTheSatoshiM` above.
```

Comment:
How do they relate? What is the purpose of having two functions? Maybe we could have a few words on this?

Reply:
Added more comments there.